.vscode/launch.json

```diff
@@ -72,7 +72,7 @@
             "request": "launch",
             "runtimeExecutable": "${execPath}",
             "args": [
-                "${workspaceFolder}/dist/src/integrationTest-samples/",
+                "${workspaceFolder}/dist/src/testFixtures/",
```

Comment:
Meant to swap this to `testFixtures/workspaceFolder` (the nesting is so https://github.com/aws/aws-toolkit-vscode/pull/1049/files#diff-3018b8392b74170c37210ba4bb10a661R22 doesn't create a file in the `src` dir). Checking real fast locally to make sure it works and will push if it does.